src/lib/redaction.js

```diff
@@ -1,10 +1,68 @@
 const redactDeath = (death, roles) => {
   return roles.includes('full-details') ? death :
     {
+      id: death.id,
       date: death.date,
+      entryNumber: death.entryNumber,
+      inquestDate: death.inquestDate,
       deceased: {
         forenames: death.deceased.forenames,
-        surname: death.deceased.surname
+        surname: death.deceased.surname,
+        dateOfBirth: death.deceased.dateOfBirth,
+        dateOfDeath: death.deceased.dateOfDeath,
+        dateOfDeathQualifier: death.deceased.dateOfDeathQualifier,
+        ageAtDeath: death.deceased.ageAtDeath,
+        birthplace: death.deceased.birthplace,
+        deathplace: death.deceased.deathplace,
+        causeOfDeath: death.deceased.causeOfDeath,
+        certifiedBy: death.deceased.certifiedBy,
+        sex: death.deceased.sex,
+        address: death.deceased.address,
+        occupation: death.deceased.occupation,
+        relationshipToPartner: death.deceased.relationshipToPartner,
+        retired: death.deceased.retired,
+        maidenSurname: death.deceased.maidenSurname,
+        aliases: death.deceased.aliases.map((e) => {
+          return {
+            type: e.type,
+            prefix: e.prefix,
+            forenames: e.forenames,
+            surname: e.surname,
+            suffix: e.suffix
+          }
```

Comment:
Is this done simply to clone the object?

If so, I don't think you really need to clone. - I'd probably just assign the existing object like everything else. (@RobinKnipe: Any thoughts on this?)

If you did still to clone there are better ways, either with `Object.assign` or spread syntax, i.e:
```js
aliases: { ...death.deceased.aliases }
```

Reply:
Sorry just realised you've got an array so you would use:
```js
aliases: [ ...death.deceased.aliases ],
```
If you wanted to clone the array.

Reply:
Yeah, not really sure what I was thinking here, changed to:

aliases: death.deceased.aliases